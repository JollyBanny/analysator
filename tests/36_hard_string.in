''''''#33'Hello, Worl'''''''#100''''''#33
