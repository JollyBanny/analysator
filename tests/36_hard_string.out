1 	 1 	 String 	 ''!Hello, Worl'''d''! 	 ''''''#33'Hello, Worl'''''''#100''''''#33
2 	 1 	 EOF 	 EOF 	 EOF